### ISSUE TITLE
General User Interface Issue

### ISSUE THREAD

**Issue by ellyasafaa** [onaio/rdt-standard#745]
Please kindly find the image below for your reference. If you need to break down the issue and creat new issue for these, please tag me on the issue.
![EoA - General User Interface](https://user-images.githubusercontent.com/73918009/106087484-afd53780-6156-11eb-88c8-8fdf0bc640b2.png)

**Comment by ellyasafaa** [comment]
@vincent-karuri @mwkhan1983

**Comment by vincent-karuri** [comment]
@ellyasafaa what is the text convention you'd like to use. I think these were copied over as was indicated from the issues.

**Comment by ellyasafaa** [comment]
@vincent-karuri Sorry Vincent, I don't get the question you asked. Could you elaborate more?

**Comment by vincent-karuri** [comment]
@ellyasafaa Could you specify what the text should be? Most of the time the text is copied as is from whatever specification you provide so please make sure it matches what is expected (including capitalization).

**Comment by vincent-karuri** [comment]
@ellyasafaa any update on this https://github.com/onaio/rdt-standard/issues/745#issuecomment-769646771?

**Comment by ellyasafaa** [comment]
@vincent-karuri 
My apologies for missing this comment.
for the capitalization, just simply capitalize the first word.

The best thing that should be fixed is the line break between the question and answer. The current interface is: the question has 2 line breaks to the answer needed, but the answer has a line break to the other (next) question. That's the point that should be fixed first.

For the text convention, I'll get back to you as soon as possible because I still need some suggestions from my colleagues.

**Comment by ellyasafaa** [comment]
@vincent-karuri Hi, Vincent. Have talked with the other testers and got some input about the interface. Please see the detail below. I think we need to open separate issues, but I'll just put it here. Could you assist me to open the new issues that will be needed accordingly?

![new client record issue](https://user-images.githubusercontent.com/73918009/106615361-f1bf0d00-659e-11eb-9ed0-820e1868cee9.png)

**Comment by vincent-karuri** [comment]
@ellyasafaa please move this https://github.com/onaio/rdt-standard/issues/745#issuecomment-771680264 to a separate issue.

**Comment by vincent-karuri** [comment]
@syedowaisali90 please fix the capitalization. Only the first character should be capitalized. Also see if you can change the spacing between question and answer fields if it's a low effort modification.

**Comment by syedowaisali90** [comment]
@ellyasafaa have you created a separate issue https://github.com/onaio/rdt-standard/issues/745#issuecomment-771680264

**Comment by syedowaisali90** [comment]
@vincent-karuri https://github.com/onaio/rdt-standard/issues/745#issuecomment-771680264 has been closed here https://github.com/onaio/rdt-standard/issues/778 can we close this one also?

**Comment by ellyasafaa** [comment]
@syedowaisali90 Yes, sure. When will I get the updated APK where this issue has been fixed?

**Comment by syedowaisali90** [comment]
@vincent-karuri the will appear on a `patient-registration` form `patient-diagnostic` form and `sample-collection` form
